docs/versioning_vendor_api.md

```diff
@@ -0,0 +1,138 @@
+# Versioning Vendor API
+
+## Summary
+
+This document describes the steps required in order to increment the minor version of the Vendor API
+
+
+## Introducing new changes
+
+Changes in the API are specified in `VersionChange` classes and each version can be made up of one or more classes. Every change class must be configured in `app/lib/vendor_api.rb`
+
+A `VersionChange` class and has a `description`, an `action` and/or one or multiple resources set using the `resource` identifier.
+
```

Comment:
Could we also add a note about updating https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/lib/vendor_api.rb#L2 which will unlock the new version being added?